### ISSUE TITLE
"Isolate by" fields under Acquisition module

### ISSUE THREAD

**Issue by rintoult** [AAFC-BICoE/dina-planning#271]
To reiterate what I said in today's meeting. 

The word "External" needs to be dropped from all the Isolated fields under the Acquisition module. 
"Isolated by" is canonical for the CCFC, equivalent in stature to "Collected By". As important and it needs the same treatment. We do not indicate under collection event "Externally Collected By". I could go into a discussion of what is External but I just want the descriptor gone.

No other user disagreed at the meeting but they are free to comment here. 

This is separate information from what might be catalogued under a preparation, however these field might for future created samples become the agent who created a preparation - but that use case for the CCFC would be rare. 

Tara

**Comment by jmacklin** [comment]
Thanks @rintoult  Glad we were able to clear this up. We had used the word "external" just to keep separate the acquisition event-based isolation versus one done as part of a preparation to generate a new material sample. The label names on these fields can always be changed. What is important though as you pointed out is that the underlying data structure is aligned with the isolated_by standard so we can share this data appropriately, and this was always the case.
